CHANGELOG.md

```diff
@@ -4,73 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.0] - 2023-06-25
+## [Unreleased]
+
+## [1.0.0] - 2024-01-14
 #### Added
-- Fields
-  - Checkbox
-  - Basic Select with icon
-  - Switch
-  - Textarea
 
 #### Changed
-- Bump Eslint & Stylelint versions and fix deprecations
-- Notification toasts usage
-- Event bus usage
-- Store: `Redux` to `Valtio`
-- `Lodash-es` to `Lodash`
-- Vite config
-
-## [1.0.0-rc.1] - 2022-09-26
-#### Added
-- Eslint
-- Stylelint
-- prettier
-- Vite bundler
-- react-router-dom
-- store
-- axios api
```

Comment:
1. `###` not `####` 
2. why u have added, changed, fixed empty under the release ?

Reply:
because it's a template

Reply:
remove added, changed, fixed then, leave just version